platformio.ini

```diff
@@ -349,6 +349,27 @@ lib_ignore    = c1921b4
   TMC26XStepper
   U8glib-HAL
 
+#
+# MKS Robin nano(STM32F103VET6)
+#
+[env:mks_robin_mini]
+platform      = ststm32
+framework     = arduino
+board         = genericSTM32F103VE
+extra_scripts = buildroot/share/PlatformIO/scripts/mks_robin_mini.py
+build_flags   = !python Marlin/src/HAL/HAL_STM32F1/STM32F1_flag_script.py
+  ${common.build_flags}
+  -DSTM32_HIGH_DENSITY
```

Comment:
i think this define is no more required... its set automatically by the sdk where its used (fsmc/timers)

Reply:
also consider the gnu++14 thing, like it was made for the SKR mini..

Reply:
I assume `-DSTM32_XL_DENSITY` is also obsolete…?

Reply:
yes..

Reply:
What's the `-g` build flag for (from the SKR Mini build environment)? Is it needed too?

Reply:
[common]
default_src_filter = +<src/*> -<src/config> -<src/HAL> +<src/HAL/shared>
build_flags = -fmax-errors=5
  -g
  -ggdb

Reply:
which are not required for firmwares.. only debug information in objects